### PR TITLE
80 cli remove spot spot from commands

### PR DESCRIPTION
**Pull request by Pbarbosa92**
No more --spot spot

### CODE REVIEW

**Review by IvanPombo — APPROVED** [review]
Confirm the behavior and let me know before merging.

**Comment by Pbarbosa92** [comment]
Working as expected!
Captura de ecrã 2024-02-01, às 09 11 36

File: inductiva/_cli/cmd_resources/cost.py
Comment (IvanPombo) [inline inductiva/_cli/cmd_resources/cost.py:1]: As you are it here: Can you round here the cost result to three decimal places and moreover add the `-n` parameter to compute the cost for `n` number of machines.

**Review by IvanPombo — APPROVED** [review]
While we are at it, please address the decimal places for the cost and add a `n` parameter for the number of machines. See my comment below.

Else...

# QA Approved

File: inductiva/resources/machine_groups.py [hunk 1]
```diff
@@ -31,7 +31,7 @@ def estimate_machine_cost(machine_type: str, spot: bool = False):
     else:
         estimated_cost = instance_price.body["on_demand_price"]
 
-    return round(float(estimated_cost), 5)
+    return round(float(estimated_cost), 3)
```
Comment (sergiomsantos) [inline inductiva/resources/machine_groups.py:34]: this is not the right place to round the values. They should be returned as is and only round when printing.

File: inductiva/_cli/cmd_resources/cost.py [hunk 2]
```diff
@@ -7,12 +7,18 @@ def estimate_machine_cost(args):
...
+    num_machines = args.num_machines
 
     cost = resources.estimate_machine_cost(
         machine_type=machine_type,
         spot=spot,
     )
-    print(f"Estimated cost of machine: {cost} $/h.")
+
+    if num_machines == 1:
+        print(f"Estimated cost of machine: {cost} $/h.")
```
Comment (sergiomsantos) [inline inductiva/_cli/cmd_resources/cost.py:18]:
```suggestion
        print(f"Estimated cost of machine: {cost:.3f} $/h.")
```

File: inductiva/_cli/cmd_resources/cost.py [hunk 3]
```diff
@@ -7,12 +7,18 @@ def estimate_machine_cost(args):
...
+    else:
+        print("Estimated total cost (per machine)"
+              f": {cost*num_machines} ({cost}) $/h.")
```
Comment (sergiomsantos) [inline inductiva/_cli/cmd_resources/cost.py:21]:
```suggestion
              f": {cost*num_machines} ({cost:.3f}) $/h.")
```

**Review by sergiomsantos — CHANGES_REQUESTED** [review]

**Pull request merged**